src/api/admin.ts

```diff
@@ -139,7 +138,6 @@ type FirestoreRoomInput_v2 = Omit<RoomInput_v2, RoomImageFileKeys> &
 
 export type PlacementInput = {
   mapIconImageFile?: FileList;
```

Comment:
Given we've removed `mapIconImageUrl`, would we not also be wanting to remove `mapIconImageFile`? Since presumably they are going to relate to the same sort of thing?